### PR TITLE
Add missing dependencies

### PR DESCRIPTION
**Pull request by fmauch**
This would be my implementation of #3

### CODE REVIEW

File: package.xml [hunk 1]
```diff
@@ -52,51 +52,35 @@
...
+  <depend>controller_interface</depend>
+  <depend>controller_manager</depend>
+  <depend>dynamic_reconfigure</depend>
+  <depend>geometry_msgs</depend>
+  <depend>hardware_interface</depend>
+  <depend>pluginlib</depend>
+  <depend>roscpp</depend>
+  <depend>speed_scaling_interface</depend>
+  <depend>trajectory_msgs</depend>
+  <depend>xacro</depend>
```
Comment (stefanscherzinger) [inline package.xml]: I'm surprised that this is not only a *test_depend*... But ok for me.

**Review by stefanscherzinger — APPROVED** [review]
That fixes things better. Let's take that.

File: package.xml (hunk 1 above)
Comment (fmauch) [reply package.xml]: I do agree.

Comment (stefanscherzinger) [reply package.xml]: @fmauch I meant the **xacro** package XD. The speed_scaling_interface is in fact a dependency..

Comment (fmauch) [reply package.xml]: Yes, I figured ;-)

**Pull request merged**